python/tests/test_seldon_client.py

```diff
@@ -50,6 +51,12 @@ def mocked_requests_post_success_json_data(url, *args, **kwargs):
     return MockResponse(json, 200, text="{}")
 
 
+def mocked_requests_post_success_raw_data(url, *args, **kwargs):
+    request = json_to_seldon_message(RAW_DATA_TEST)
+    json = seldon_message_to_json(request)
```

Comment:
Won't the `json` variable match what's in `RAW_DATA_TEST`? Also, since `mocked_requests_post_success_raw_data` only seems to mock the response, can't we use one of the existing mock side effects?

Reply:
1. regarding json variable: did same implementation as in mocked_requests_post_success_json_data. this is done multiple times in the client code also - I understand from Alejandro that it is a kind of typing checks / make sure... ( ofcource itis not so relevant in unit-testings)
2. mocked_requests_post_success_json_data can be use - but the return value will be of JSON_TEST_DATA. I defined a new one for competible.

Reply:
will change to use moked....post_success